kubemarine/plugins/__init__.py

```diff
@@ -132,29 +123,28 @@ def verify_image_redefined(plugin_name, previous_version, base_plugin, cluster_p
 def install(cluster, plugins=None):
     if not plugins:
         plugins = cluster.inventory["plugins"]
-    plugins_queue = []
+    plugins_queue: List[str] = []
     max_priority = 0
     for plugin_name, plugin_item in plugins.items():
         if plugin_item.get("install", False) and plugin_item.get("installation", {}).get('procedures') is not None:
-            plugin_item['plugin_name'] = plugin_name
```

Comment:
This is for generation of valid cluster_finalized. See test_schema.py/FinalizedInventoryValidation.test_plugins_installation_enriches_valid